tests/test_pager.py

```diff
@@ -0,0 +1,85 @@
+import unittest
+import recurly
+from recurly.pager import Pager
+from recurly import Resource
+from .mock_resources import MyResource, MySubResource
+from .mock_client import MockClient
+import unittest.mock as mock
+from unittest.mock import Mock, MagicMock
+
+
+def get_pager_client():
+    conn = MagicMock()
+    conn.request = MagicMock(return_value=None)
+    conn.getresponse = MagicMock()
+    conn.getresponse.side_effect = [first_page(), second_page()]
```

Comment:
This means that each time you call this method, it will return the next item in the list. So the first time getresponse is called, the first page's mock response is returned, the second time, the last page mock response is returned.